src/full_node/blockchain.py

```diff
@@ -80,6 +80,8 @@ class Blockchain:
     block_store: BlockStore
     # Coinbase freeze period
     coinbase_freeze: uint32
+    # Map (challenge_hash, iters) to height.
+    pot_to_chain_height: Dict[Tuple[bytes32, uint64], uint32]
```

Comment:
It is inconsistent to store this map here, and to also store proof of time heights in full node store

Reply:
so this should belong to block_store, right?

Reply:
Yes that makes sense